docs(javascript): update sentry-testkit page (#6445)
getsentry/sentry-docs: src/platforms/javascript/common/configuration/sentry-testkit/index.mdx

When building tests for your application, you want to assert that the right flow-tracking or error is being sent to Sentry, but without really sending it to the Sentry servers. This way you won't swamp Sentry with false reports during test runs or other CI operations.

> [Sentry Testkit](https://zivl.github.io/sentry-testkit/) is a Sentry plugin that allows intercepting Sentry's reports and further inspection of the data being sent. It enables Sentry to work natively in your application, and by overriding the default Sentry's transport mechanism, the report is not really sent, but rather logged locally into the memory. This way, logged reports can be fetched later for your own usage, verification, or any other use you may have in your local developing/testing environment.

Reviewer: ```suggestion
[Sentry Testkit](https://zivl.github.io/sentry-testkit/) is a Sentry plugin that allows Sentry's reports to be intercepted for further data inspection. It enables Sentry to work natively in your application, by overriding Sentry's default transport mechanism, which makes it so that the report isn't really sent, but rather logged locally into memory. This way, logged reports can be fetched later for your own usage, verification, or any other purpose you may have in your local developing or testing environment.
```

Author: Thanks!)

  dsn: "___PUBLIC_DSN___",
  transport: sentryTransport,
  //... other configurations

> })

Author: ```suggestion
});
```
